SDWebImage/SDMemoryCache.m

```diff
@@ -54,15 +46,11 @@ - (void)commonInit {
     } else {
         self.memCache = [[SDMemoryNSCache alloc] initWithConfig:self.config];
     }
-    
-    SDImageCacheConfig *config = self.config;
-    self.totalCostLimit = config.maxMemoryCost;
-    self.countLimit = config.maxMemoryCount;
-    
-    [config addObserver:self forKeyPath:NSStringFromSelector(@selector(maxMemoryCost)) options:0 context:SDMemoryCacheContext];
-    [config addObserver:self forKeyPath:NSStringFromSelector(@selector(maxMemoryCount)) options:0 context:SDMemoryCacheContext];
 }
 
+- (void)setObject:(nullable id)object forKey:(nonnull id)key {
```

Comment:
We should implement  `<SDMemoryCache>` all method.

Reply:
LGTM